src/accounts/api/serializers.py

```
@@ -102,14 +102,18 @@ class Meta:
     email = serializers.EmailField()
 
     def validate(self, data):
-        account = Account.objects.filter(email=data["email"]).first()
-        #note: make this silently fail later, users dont need to know about this.
+        email = data.get("email")
+        account = Account.objects.filter(email=email).first()  # Assuming you're using the User model
+        # Note: Make this silently fail later; users don't need to know about this.
         if account is None:
             raise serializers.ValidationError("Account with this email doesn't exist.")
-        newModel = PasswordResetRequestModel(token=account.verification_token, account=account)
-        return newModel
+
+        # Create a new instance of PasswordResetRequestModel using the account's verification token
+        token = account.verification_token  # Replace with the actual field name
```

This is the wrong token.

---

token uses uuid for now, should convert to `secrets.token_urlsafe(32)` but it keeps breaking the thingy majingy and silently fails